guides/source/getting_started.md

```diff
@@ -191,6 +191,7 @@ of the files and folders that Rails creates by default:
 |.dockerignore|This file tells Docker which files it should not copy into the container.|
 |.gitattributes|This file defines metadata for specific paths in a git repository. This metadata can be used by git and other tools to enhance their behavior. See the [gitattributes documentation](https://git-scm.com/docs/gitattributes) for more information.|
 |.gitignore|This file tells git which files (or patterns) it should ignore. See [GitHub - Ignoring files](https://help.github.com/articles/ignoring-files) for more information about ignoring files.|
+|.rubocop.yml|This file contains the configuration for RuboCop, a omakase ruby styling for rails.|
```

Comment:
```suggestion
|.rubocop.yml|This file contains the configuration for RuboCop.|
```

I don't think we should include the specifics of the file contents. Looking at similar files: `.dockerignore` and `.gitignore` explain _what_ they do (ignore files) but not _which_ files they ignore.

Reply:
Thanks @skipkayhil, Updated the change